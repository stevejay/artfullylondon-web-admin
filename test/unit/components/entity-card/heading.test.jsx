import React from 'react'
import { shallow } from 'enzyme'

import EntityCardHeading from '_admin/components/entity-card/heading'

it('should render correctly', () => {
  const wrapper = shallow(
    <EntityCardHeading id='some-id'>
      <div id='child' />
    </EntityCardHeading>
  )

  expect(wrapper).toMatchSnapshot()
})
