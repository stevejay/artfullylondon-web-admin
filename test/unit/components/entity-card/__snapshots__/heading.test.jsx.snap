// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<h3
  className="components-entity-card-heading-m__heading"
  id="some-id"
>
  <div
    id="child"
  />
</h3>
`;
